apps/qubit/modules/informationobject/actions/boxLabelCsvAction.class.php

```diff
@@ -38,7 +38,7 @@ public function execute($request)
     $csv = fopen('php://temp/maxmemory:'. (2*1024*1024), 'r+');
 
     // Write CSV header
-    fputcsv($csv, array('referenceCode', 'physicalObjectName', 'title', 'creationDates'));
+    fputcsv($csv, array('referenceCode', 'physicalObjectName', 'title', 'displayDates'));
```

Comment:
Ops, I will fix that. That was me misunderstanding things 😅
On Mar 30, 2016 5:28 PM, "MikeFE" <notifications@github.com> wrote:

> In
> apps/qubit/modules/informationobject/actions/boxLabelCsvAction.class.php
> <https://github.com/artefactual/atom/pull/306#discussion_r57985994>:
>
> > @@ -38,7 +38,7 @@ public function execute($request)
> >      $csv = fopen('php://temp/maxmemory:'. (2*1024*1024), 'r+');
> >
> >      // Write CSV header
> > -    fputcsv($csv, array('referenceCode', 'physicalObjectName', 'title', 'creationDates'));
> > +    fputcsv($csv, array('referenceCode', 'physicalObjectName', 'title', 'displayDates'));
>
> I'm a bit unclear here, why are we changing the CSV column name?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/artefactual/atom/pull/306/files/28ba43a97a077bde29701e9862a96ae2ad944760#r57985994>
>